site/content/extend/plugins/community-plugin-marketplace.md

```diff
@@ -0,0 +1,91 @@
+6. After the PR is merged, the plugin gets promoted across Mattermost social media and swag sent to the maintainer. If there are multiple maintainers, everyone gets swag.
+
+### Process for updating community plugin to the Marketplace
+1. Open an issue on the Plugin Marketplace Repository using [a pre-defined template for new plugins](https://github.com/mattermost/mattermost-marketplace/issues/new?template=update_plugin.md). The template contains the checklist above, so you can check the items. Please also point out which commit should be used for the review. You may cut a release candidate (RC) for the reviews.
+2. The reviews are requested by the _owner_. This is done by cloning the following JIRA ticket for every reviewer, assigning it and mentioning the ticket on GitHub. The reviewers point out discovered issues on the bug tracker of the community plugin. Once all issues are resolved, they comment on the issue stating the positive review and mark the JIRA ticket as resolved.
+    - [PM](https://mattermost.atlassian.net/browse/MM-22228)
+    - [Dev](https://mattermost.atlassian.net/browse/MM-22222)
+    - [Security](https://mattermost.atlassian.net/browse/MM-22226)
+    - [QA](https://mattermost.atlassian.net/browse/MM-22227)
+3. Once all items are checked and reviews made, the fork under the Mattermost GitHub organization is updated to the commit used for review.
+```sh
+git fetch upstream
+git checkout master
+git merge --ff-only $COMMIT_HASH
+```
+4. `/mb cutplugin --repo $REP --tag $TAG` is run to build, sign and upload the approved commit of the plugin.
+5. The _owner_ opens a PR, which adds the plugin to `plugins.json` using `generator add $REP $TAG --community`. Only a functional review by one dev and one QA member is needed for this PR.
+6. Promotion via social media might happen on outstanding updates.
+
+### Beta plugins
+If a community plugin doesn’t make it through the review process, it may still be added to the Marketplace and marked as “Beta” via a label. The reviewers make a case by case decision if the quality of a plugin is sophisticated enough to be added to the Marketplace. Security and functional reviews and Items 1, 2, 3 and 5 from the [Product Requirements Checklist](#requirements-for-adding-community-plugin-to-the-marketplace) must be fulfilled for beta plugins.
+
+It must be made clear in the Marketplace UI that a plugin is in beta. All beta plugins must only be visible on Mattermost Servers that support labels, which is Mattermost v5.20.
+
+### Take down Policy
+If an S2 and above security issue or bug that prevents the usage of the plugin for many users is not fixed within 14 days, the plugin gets taken from the Marketplace. It may be resubmitted, once the issue is resolved. Mattermost reserves the right to take down plugins at any time if a fix for a security issue is not forthcoming or the issue is critical enough to justify an immediate takedown.
```

Comment:
What is an S2?

Also, some minor phrasing change suggestions:
> [...] the plugin will be removed from the Marketplace. It may be resubmitted once the issue is resolved.

Reply:
S2 is a classification level used my Mattermost for security issues.

@jupenur Do we have public documentation about our classification system?

Reply:
I don't think we do, but we're also in the process of moving away from the S1/S2/S3 system to the more standard [CVSS](https://www.first.org/cvss/calculator/3.0).

An S2 would correspond to a **medium** (4.0 and above) CVSS base score. The wording should probably be changed to use the latter. We'll likely have made the transition by the time this is live, and even if we haven't, "medium severity" is easier to understand and still maps nicely back to the old system.